src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -147,7 +154,8 @@ const TrackPanelListItem = (props: TrackPanelListItemProps) => {
 
   const listItemClassNames = classNames(styles.listItem, {
     [styles.main]: track.track_id === 'gene',
-    [styles.currentDrawerView]: track.track_id === drawerView
+    [styles.highlightTrack]:
```

Comment:
Will need to change the class name here if it is changed in `TrackPanelListItem.scss`

Reply:
That's done